pkg/kubectl/resource/builder.go

```diff
@@ -602,6 +602,22 @@ func (b *Builder) SingleResourceType() *Builder {
 // prefers a fully specified GroupVersionKind match. If we don't have one, match on a fully specified
 // GroupVersionResource, or fallback to a match on GroupResource.
```

Comment:
Given the current changes the godoc needs update as well.

Reply:
Thanks for catching that. Updated godoc to reflect new changes